Dockerfile

```diff
@@ -13,9 +13,9 @@ RUN if [ $(uname -p) != x86_64 ]; then sed -i "s/^debug = false$/debug = false\n
 RUN mkdir -p .cargo && echo -e '[net]\ngit-fetch-with-cli = true' > .cargo/config.toml
 RUN cargo build --release
 
-FROM registry.fedoraproject.org/fedora:41
-RUN dnf install -y /usr/bin/gpg /usr/sbin/kpartx /usr/bin/lsblk \
-    /usr/sbin/udevadm && \
+FROM quay.io/fedora/fedora:42
+RUN dnf install -y /usr/bin/gpg /usr/bin/kpartx /usr/bin/lsblk \
+    /usr/bin/udevadm && \
     dnf clean all
-COPY --from=builder /build/target/release/coreos-installer /usr/sbin
+COPY --from=builder /build/target/release/coreos-installer /usr/bin
```

Comment:
you need to change the below line to `bin` as well

Reply:
updated it now